news/_posts/releases/2018-10-03-prestashop-1-7-4-3-1-6-1-21-maintenance-releases.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: post
-title:  "Release of PrestaShop 1.7.4.3 and 1.6.1.21"
+title:  "Release of PrestaShop 1.7.4.4 and 1.6.1.23"
```

Comment:
You are updating a file rather than creating a new one. Is it intended ?

Reply:
I think I perhaps did a mistake. Let me check.